### PR TITLE
DOC-3416: Release Notes for Oct 18 release

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3416](https://openedx.atlassian.net/browse/DOC-3416)

Add a description of your changes with links to any relevant material.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] ORA item: @efischer19  
- [x] Verification item: @schenedx 
- [x] Analytics item: @dsjen 
- [x] CAPA item: @sstack22 
- [x] Doc team review: @srpearce 
- [x] Product review: @marcotuts @sstack22 

FYI: @jaakana, @mmacfarlane @dhixonedx 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors - with exception of known issue caused by xrefs not yet resolvable to new anchors.
### DRAFT HTML Output for Reviewers
- [x] draft-release-notes.readthedocs.io/en/latest/2016/2016-10-18.html
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2016/lms/lms_2016-10-18.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+* In open response assessments, when you override a learner's grade by
+  performing a staff assessment, any steps that the learner did not complete
+  for the assignment are now marked as complete. Previously, steps that
+  required action from the learner remained open and the assignment was left
+  in an incomplete state. (:jira:`TNL-4696`)
+
+  For more information about overriding a learner's grade in an ORA assignment,
+  see :ref:`partnercoursestaff:Override a learner assessment grade`.
+
+* For courses that require ID verification, learners can now submit
```
Comment (schenedx) [inline en_us/release_notes/source/2016/lms/lms_2016-10-18.rst]: @catong This is not true with what we have in Stage.
Should be something in the line of: "Learners do not need to verify again when their current approved ID verification is not expired"

**Review by dsjen — APPROVED** [review]
LGTM! 👍

**Comment by sstack22** [comment]
👍 !

File: en_us/release_notes/source/2016/lms/lms_2016-10-18.rst (hunk 1 above)
Comment (catong) [reply en_us/release_notes/source/2016/lms/lms_2016-10-18.rst]: @schenedx I must have misunderstood, but your PR https://github.com/edx/edx-platform/pull/13578 has this description: "Updated photo verification view to accept photos if verification is set to expire soon." Isn't "Learners do not need to verify again when their current approved ID verification is not expired" the unchanged (current) behavior?

**Comment by catong** [comment]
@schenedx I updated the description - please take a look. Thanks!

File: en_us/release_notes/source/2016/lms/lms_2016-10-18.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+* In open response assessments, when you override a learner's grade by
+  performing a staff assessment, any steps that the learner did not complete
+  for the assignment are now marked as complete. Previously, steps that
+  required action from the learner remained open and the assignment was left
+  in an incomplete state. (:jira:`TNL-4696`)
+
+  For more information about overriding a learner's grade in an ORA assignment,
+  see :ref:`partnercoursestaff:Override a learner assessment grade`.
+
+* For courses that require ID verification, learners who have valid
```
Comment (schenedx) [inline en_us/release_notes/source/2016/lms/lms_2016-10-18.rst:10]: 👍

**Review by schenedx — APPROVED** [review]
👍

File: en_us/release_notes/source/2016/lms/lms_2016-10-18.rst (hunk 1 above)
Comment (schenedx) [reply en_us/release_notes/source/2016/lms/lms_2016-10-18.rst]: @catong With the verification issue I encountered yesterday, the behavior with this release has changed from what's in the PR to what you got right now. It's very confusing. And thank you for the updated message

File: en_us/release_notes/source/2016/lms/lms_2016-10-18.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+* In open response assessments, when you override a learner's grade by
+  performing a staff assessment, any steps that the learner did not complete
+  for the assignment are now marked as complete. Previously, steps that
+  required action from the learner remained open and the assignment was left
+  in an incomplete state. (:jira:`TNL-4696`)
+
+  For more information about overriding a learner's grade in an ORA assignment,
+  see :ref:`partnercoursestaff:Override a learner assessment grade`.
```
Comment (efischer19) [inline en_us/release_notes/source/2016/lms/lms_2016-10-18.rst:8]: 👍

**Review by efischer19 — APPROVED** [review]

File: en_us/release_notes/source/2016/studio/studio_2016-10-18.rst [hunk 4]
```diff
@@ -0,0 +1,9 @@
+  This release includes an enhancement to the core checkbox and multiple choice
+  problem types. When you edit these problems in the advanced editor, you can
+  now set attributes and options by adding a ``<script>`` element.
+
+  For more information, see :ref:`partnercoursestaff:Using the Script Element
+  in Checkbox Problems` or :ref:`partners:Using the Script Element in Multiple
```
Comment (srpearce) [inline en_us/release_notes/source/2016/studio/studio_2016-10-18.rst]: Should this ref prefix also be partnercoursestaff?

**Review by srpearce — APPROVED** [review]
Just one comment.

File: en_us/release_notes/source/2016/studio/studio_2016-10-18.rst (hunk 4 above)
Comment (catong) [reply en_us/release_notes/source/2016/studio/studio_2016-10-18.rst]: Thank for catching that!

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1316

**Pull request merged**